SyncIntegrationTests/tps.py

```diff
@@ -27,7 +27,7 @@ def run(self, test, phase='phase1', ignore_unused_engines=True):
             'testing.tps.testPhase': phase,
             'testing.tps.ignoreUnusedEngines': ignore_unused_engines,
         })
-        args = ['-marionette']
+        args = ['-marionette','-headless']
```

Comment:
Mark from sync told me a few months ago that we could use the headless mode for our purposes so that there's no Firefox pop-up.